### ISSUE TITLE
Keybase will not let me send any messages on any devices

### ISSUE THREAD

**Issue by mg10011** [keybase/keybase-issues#3849]
I get the following error anytime I write a message on any Keybase chat:

`This message failed to send.  Cancel or Retry.`

I have tried everything and there seems to be no documentation for this error that several other users have had.  I have also done the following things, as per Keybase's own advice for greater security.

1.  I have installed Keybase on several other devices (4 computers and 1 phone).
2.  I have proofed my Keybase account with Twitter and Reddit accounts.

WTF?

**Comment by maxtaco** [comment]
Keybase username?

**Comment by mg10011** [comment]
Username: mg10111

**Comment by maxtaco** [comment]
Someone will investigate

**Comment by Martial83** [comment]
Hi,
I have exactly the same problem.
Have you found a resolution?
Thank's

**Comment by Martial83** [comment]
The problem was with windows! 
I killed all the windows process and the problem is gone.
Sorry ...

**Comment by j2apolo** [comment]
Same here, will someone sort out it? I tried all, Uninstalled the app, installed again but nothing.

Its says 

This message failed to send. Cancel or Retry.

![Screenshot_20200510-133154](https://user-images.githubusercontent.com/34243052/81494450-a9e8af00-92c6-11ea-8d1a-778eb6a057e2.png)

**Comment by CryptoShibe** [comment]
I have this same issue but it's only with one account.
From one account only, I can't chat from:
1) My Mac
2) Ubuntu Lab Machine
3) iPhone 12

From all of these devices, my other keybase accounts work normally. 
No errors when I attempt to chat.

It's almost exactly like... I've been shadow banned. Is that a thing?

**Comment by arvidn** [comment]
I'm also experiencing this, and I'm really disappointed that the error message doesn't contain *any* hint of what's failing.

**Comment by dkvasnicka** [comment]
Same here. I can receive messages but I'm not able to send anything on any device. Username same as here.

**Comment by itoldusoandso** [comment]
Exactly. Same issue. Can't send message from any device, I tried from Android, Mac, iOS.

**Comment by dkvasnicka** [comment]
Updating the app solved the issue for me.
